config.py

```diff
@@ -2547,13 +2547,17 @@ def _set_eeg_montage(cfg, raw, subject, session) -> None:
 
     Modifies ``raw`` in-place.
     """
-    montage_name = cfg.eeg_template_montage
-    if cfg.datatype == 'eeg' and montage_name:
+    montage = cfg.eeg_template_montage
+    is_mne_montage = isinstance(montage,
+                                mne.channels.montage.DigMontage)
+    montage_name = ('custom_montage' if is_mne_montage else montage)
```

Comment:
```suggestion
    montage_name = 'custom_montage' if is_mne_montage else montage
```

Reply:
It can help readability to have the parenthesis, often done in MNE code.

Reply:
I've honestly never seen this in MNE 🤔 
If you're concerned about readability, shouldn't we do
```python
if is_mne_montage:
    montage_name = 'custom_montage'
else:
    montage_name = montage
```
?

Reply:
do as you prefer, I don't care very much. I find my one-line a good compromise though :)